tests/memory_checker/test_memory_checker.py

```diff
@@ -36,6 +38,12 @@ def modify_monit_config_and_restart(duthost):
     Returns:
         None.
     """
+    dut_name, container_name = decode_dut_port_name(enum_dut_feature_container)
```

Comment:
```decode_dut_port_name```? Is it a typo? ```decode_dut_port_name``` doesn't return dut_name and container name.

Reply:
Oh, I see. It's extremely tricky. I don't suggest that usage although dutname|container_name has exactly same format as dutname|portname. Can you please add another interface like ```decode_dut_container_name```?

Reply:
Good suggestion! Will update.

Reply:
Updated!